src/main/java/org/jenkinsci/plugins/microsoft/appservice/commands/DefaultDockerClientBuilder.java

```diff
@@ -27,6 +30,12 @@ public DockerClient build(AuthConfig authConfig) {
                 .withRegistryUrl(authConfig.getRegistryAddress())
                 .withRegistryEmail(authConfig.getEmail());
 
+        // must enable tcp on windows by check the option "Expose daemon on tcp://localhost:2375 without TLS"
+        // more reading at https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/configure-docker-daemon
+        if (SystemUtils.IS_OS_WINDOWS) {
```

Comment:
Does this check run on master? What if user has a Linux master but runs job on Windows slaves?